src/datasets/config.py

```diff
@@ -191,6 +191,9 @@
 MAX_DATASET_CONFIG_ID_READABLE_LENGTH = 255
 
 # Streaming
-
 STREAMING_READ_MAX_RETRIES = 20
 STREAMING_READ_RETRY_INTERVAL = 5
+
+# Datasets without script
+DATA_FILES_MAX_NUMBER = 10
+ARCHIVED_DATA_FILES_MAX_NUMBER = 5
```

Comment:
Maybe have more explicit variable names ? It's ok if they're long
```suggestion
DATA_FILES_MAX_NUMBER_FOR_MODULE_INFERENCE = 10
ARCHIVED_DATA_FILES_MAX_NUMBER_FOR_MODULE_INFERENCE = 5
```